src/main/java/uk/ac/ebi/pride/utilities/data/controller/DataAccessUtilities.java

```diff
@@ -444,39 +438,39 @@ public static int getNumberOfSubstitutionPTMs(Peptide peptide) {
         return cnt;
     }
 
-    /**
-     * Get peptide score from a peptide object.
-     *
-     * @param paramGroup parameter group
-     * @param seTypes    a list of search engine types
-     * @return PeptideScore  peptide score
-     */
-    public static Score getPeptideScore(ParamGroup paramGroup, List<SearchEngineType> seTypes) {

```

Comment:
@noedelta We need to fine if this function is not breaking the inspector and PIA